### PR TITLE
UI updates to support the packages page

### PR DESCRIPTION
**Pull request by cnunciato**
Accompanies https://github.com/pulumi/registry/pull/25.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
     "version": "0.0.1",
     "description": "The Hugo theme of the Pulumi website.",
     "private": true,
-    "license": "MIT",
-    "dependencies": {
+    "license": "Apache-2.0",
+    "devDependencies": {
```
Comment (praneetloke) [inline package.json:7]: Why the change to `devDependencies`?

**Review by praneetloke — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (cnunciato) [reply package.json:12]: You're right, this doesn't quite capture it. I've updated to pull only the "real" dependencies into `dependencies`, and leave the rest as `devDependencies`.
